src/Microsoft.AspNetCore.SignalR.Common/Internal/Protocol/IHubProtocol.cs

```diff
@@ -19,6 +19,8 @@ public interface IHubProtocol
 
         void WriteMessage(HubMessage message, IBufferWriter<byte> output);
 
+        byte[] GetBytes(HubMessage message);
```

Comment:
I wonder about this name. Since we have `WriteMessage`, should this be `GetMessageBytes`?

Reply:
Done